File: proxy/tests/script_engine/req_change_query.lua
Extensions = {
	{
		request = function(flow)
			flow.request.url.searchParams["foo"] = "bar"
			flow.request.url.searchParams["a"] = "b"
			flow.request.url.searchParams["no"] = nil
			flow.request.url.searchParams["yes"] = nil
			flow.request.url.searchParams["saison"] = nil
		end,
	},
}

File: proxy/tests/script_engine/req_change_query.js
globalThis.Extensions = [{
  request(flow) {
    flow.request.url.searchParams.set("foo", "bar");
    flow.request.url.searchParams.set("a", "b");
    flow.request.url.searchParams.delete("no");
    flow.request.url.searchParams.delete("yes");
    flow.request.url.searchParams.delete("saison");
  }
}];
